plugins/com.google.cloud.tools.eclipse.appengine.newproject.maven.test/src/com/google/cloud/tools/eclipse/appengine/newproject/maven/MavenArchetypeProjectWizardTest.java

```diff
@@ -16,25 +16,36 @@
 
 package com.google.cloud.tools.eclipse.appengine.newproject.maven;
 
+import static org.junit.Assert.assertEquals;
 import static org.junit.Assert.assertNotNull;
+import static org.mockito.Mockito.mock;
 
+import org.eclipse.jface.wizard.IWizardContainer;
 import org.eclipse.swt.widgets.Display;
+import org.eclipse.swt.widgets.Shell;
+import org.junit.After;
 import org.junit.Assert;
 import org.junit.Before;
 import org.junit.Test;
 
 public class MavenArchetypeProjectWizardTest {
 
   private MavenArchetypeProjectWizard wizard;
+  private Shell shell;
 
   @Before
   public void setUp() {
     assertNotNull(Display.getDefault());
-
+    shell = new Shell(Display.getDefault());
```

Comment:
Thanks for the reference. It's good to learn new things. Since the setup and tear-down processes for this test is very small, I'll go with simplicity here.